Source/ARTStatus.m

```diff
@@ -9,12 +9,14 @@
 #import <Foundation/Foundation.h>
 
 #import "ARTStatus.h"
+#import "ARTNSError+ARTUtils.h"
 
 // Reverse-DNS style domain
 NSString *const ARTAblyErrorDomain = @"io.ably.cocoa";
 
 NSString *const ARTErrorInfoStatusCodeKey = @"ARTErrorInfoStatusCode";
 NSString *const ARTErrorInfoOriginalDomainKey = @"ARTErrorInfoOriginalDomain";
+NSString *const ARTErrorInfoRequestIdKey = @"request_id";
 
```

Comment:
Two keys above have Pascal Case notation, and this suddenly with underscore.

Reply:
changed here aed56c3b4503b680c2469a4037e6aa0ec92e70a6